docs/src/main/asciidoc/se/jsonrpc/client.adoc

```diff
@@ -0,0 +1,144 @@
+
+    Copyright (c) 2025 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= JSON-RPC Client
+:description: Helidon JSON-RPC Client
+:keywords: helidon, json-rpc, jsonrpc, client, java, se
+:feature-name: WebClient/JSON-RPC
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/se.adoc[]
+
+== Contents
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+** <<Simple Requests, Simple Requests>>
+** <<Batch Requests, Batch Requests>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+
+== Overview
+
+
+The Helidon JSON-RPC client API is part of the WebClient API, and can be used to
+create link:https://www.jsonrpc.org/specification[JSON-RPC 2.0] client
+applications. It offers built-in support to invoke JSON-RPC server methods with
+minimal effort, including handling of JSON parameters
+and processing of JSON responses.
+
+include::{rootdir}/includes/dependencies.adoc[]
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.webclient</groupId>
+    <artifactId>helidon-webclient-jsonrpc</artifactId>
+</dependency>
+----
+
+== Usage
+
+=== Simple Requests
+
+An instance of `JsonRpcClient` can be obtained from a configured `WebClient`
+instance as shown next:
+
+[source,java]
+----
+include::{sourcedir}/se/jsonrpc/ClientSnippets.java[tag=snippet_1, indent=0]
+----
+
+The `WebClient` instance is configured with a base URI and the `JsonRpcClient`
+instance is created from it passing the `JsonRpcClient.PROTOCOL` selector.
+
+To create a request, simply pass the method name, the ID and some
+parameters using the fluent API provided. Parameters must be JSON values, but
+simple Java types such as `String` and `int` are supported and mapped
+to the corresponding JSON types automatically.
+
+[source,java]
+----
+include::{sourcedir}/se/jsonrpc/ClientSnippets.java[tag=snippet_2, indent=0]
+----
+
+A `JsonRpcClientResponse` is a subtype of `HttpClientResponse`, so any methods
+available in the latter apply to the former. Thus, we can easily verify the
+HTTP status and then inspect if any JSON-RPC result has been returned as
+follows:
+
+[source,java]
+----
+include::{sourcedir}/se/jsonrpc/ClientSnippets.java[tag=snippet_3, indent=0]
+----
+
+NOTE: If the JSON-RPC response contains an error, it will also contain
```

Comment:
Our style guide urges using the present tense. So: "...error, it contains..." instead, for example.

There are some other places elsewhere using future tense as well.

Reply:
Fixed